### PR TITLE
fixes for 15133

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15133

*Description of changes:*
- Text fix for 15133
- Fixed autoscroll for Transfer Review and Confirm from 15133
- Fixed Transfer payload filing schema error with phone number
- Fixed radio button becoming disabled bug from 15133

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -715,11 +716,37 @@ export default defineComponent({
...
+      }
+      // Set the first invalid section
+      const invalidSection = document.getElementById(mappedSections.filter(item => item.value === false)[0].section)
+
       setTimeout(() => {
         scrollToTop
           ? document.getElementById('mhr-information-header').scrollIntoView({ behavior: 'smooth' })
           : document.getElementsByClassName('border-error-left').length > 0 &&
-            document.getElementsByClassName('border-error-left')[0].scrollIntoView({ behavior: 'smooth' })
+            invalidSection.scrollIntoView({ block: 'start', inline: 'nearest', behavior: 'smooth' })
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: I'm not sure this is required.

What is the requirement/issue for this code?

Comment (owillborn) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: The autoscroll was scrolling to the top of the components, not the section. This enables it to scroll to where the number header is. Ill pm you some screenshots

**Review by doug-lovett — APPROVED** [review]
LGTM.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 2]
```diff
@@ -556,6 +557,15 @@ export default defineComponent({
       /** True if Jest is running the code. */
       isJestRunning: computed((): boolean => {
         return process.env.JEST_WORKER_ID !== undefined
+      }),
+      sectionValidationStates: computed((): Array<any> => {
```
Comment (owillborn) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: this should be Array of boolean will change

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 3]
```diff
@@ -589,6 +599,24 @@ export default defineComponent({
...
+      ]
+      const mappedSections = validationSections.map(function (key, val) {
+        return { section: key, value: localState.sectionValidationStates[val] }
+      })
+      if (!isRoleStaffReg.value) {
+        mappedSections.shift()
+        mappedSections.pop()
+      }
+      return mappedSections
+    }
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Should this be computed as well and include/return
`const invalidSection = document.getElementById(mappedSections.filter(item => item.value === false[0].section)`?

Then you could just pass this value (`getInvalidSection`) or whatever into your scrollToSection method?

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 4]
```diff
@@ -715,11 +743,15 @@ export default defineComponent({
...
+      const mappedSections = mapTransferReviewValidations()
+      // Set the first invalid section
+      const invalidSection = document.getElementById(mappedSections.filter(item => item.value === false)[0].section)
+
       setTimeout(() => {
         scrollToTop
           ? document.getElementById('mhr-information-header').scrollIntoView({ behavior: 'smooth' })
           : document.getElementsByClassName('border-error-left').length > 0 &&
-            document.getElementsByClassName('border-error-left')[0].scrollIntoView({ behavior: 'smooth' })
+            invalidSection.scrollIntoView({ block: 'start', inline: 'nearest', behavior: 'smooth' })
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: I don't think you will need the short circuit protection here anymore, with the refactors.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 3 above)
Comment (owillborn) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Sure, I'll do that to keep the methods completely separate

**Review by cameron-eyds — APPROVED** [review]
LGTM though! Other than the comments i've already made 👍

**Review by cameron-eyds — APPROVED** [review]

**Pull request merged**